platform-sdk/docs/proposals/001-tss-rosters-design-proposal/proposal.md

```diff
@@ -524,16 +604,23 @@ Some of the obvious test cases to be covered in the plan include validating one
    that is still active on the network when the node comes back online. Verify it can still successfully rejoin the
    network.
 
-## Metrics
-
-We propose that some metrics be added. One useful metric we will introduce is the number of candidate rosters that have
-been set. Others may be introduced during implementation.
-
 ## Implementation and Delivery Plan
 
 - Define Roster Data Structure in protobuf: Design and implement the Roster class and the associated child classes.
 - Develop the API: Method added and implemented in `PlatformState` to set a candidate roster.
 - Modify State Storage: Implement specified data structures and candidate roster logic.
-- Update Components: Modify all platform components that currently use `AddressBook` to use `Rosters` instead, if any.
+- Update Components: Modify all current platform components that currently use `AddressBook` to use `Rosters` instead,
+  if any.
 - Testing: Conduct thorough testing as specified in this proposal.
 - Deployment: Work with DevOps and Release Engineering to deploy the changes.
+
+### DevOps Changes, in summary
+
+1. DevOps will decide whether to keep the `config.txt` model and if so, work with Services on its translation into a
+   Roster.
+2. DevOps will be responsible for providing the `config.txt`, `genesis-config.txt` or `override-config.txt` as files in
+   the desired location.
+3. DevOps will be responsible for managing the lifecycle of the files on disk that are used in creating the Roster,
+   including the new TSS encryption key.
+4. DevOps will be responsible for cleaning up the `override-config.txt` file after the network transplant process is
+   complete.
```

Comment:
No strong opinion here, but we could consider the `.archive/<date_time>/` approach here to simplify DevOps' life. If we don't, then this may cause issues when e.g. one restarts a node twice with an override roster present - they'll end up completely forgetting the actual previous roster.

Speaking of which, did we consider all the edge-cases for PCES replay when an override roster is provided? Are we sure we can safely forget the actual previous roster in any and all cases?

Reply:
Agreed, this was a section I hadn't yet got to updating.